rend3-egui/src/lib.rs

```diff
@@ -171,7 +169,7 @@ impl EguiRenderRoutine {
             texture_size,
         );
 
-        egui_wgpu_backend::RenderPass::egui_texture_from_wgpu_texture(
+        egui_wgpu::renderer::RenderPass::register_native_texture(
```

Comment:
This is the offending function. It was removed from egui when migrating from `egui_wgpu_backend` to `egui-wgpu`, then brought back a few weeks after the release of 0.18.